### PR TITLE
AC-842 fixed the repository layer for AddEditPatient package

### PR DESCRIPTION
**Pull request by LuGO0**
Added the Missing repository layer abstraction for 3 presenter methods.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-842

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by LuGO0** [comment]
@rishabh-997 please review !!

**Comment by LuGO0** [comment]
@rishabh-997 there seems to be some problem with the git history of the changed files, I am trying to see what caused this .

**Comment by LuGO0** [comment]
> @rishabh-997 there seems to be some problem with the git history of the changed files, I am trying to see what caused this .

The git history is fixed now @rishabh-997 now its ready for review !!

**Review by debarghya472 — APPROVED** [review]
looks ok!

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientPresenter.java [hunk 1]
```diff
@@ -272,28 +274,25 @@ public PlacesClient getPlaces() {
...
-            public void onResponse(Call<Results<SystemProperty>> call, Response<Results<SystemProperty>> response) {
-                if (response.isSuccessful()) {
-                    String uuid = response.body().getResults().get(0).getConceptUUID();
-                    if (uuid.length() == 36) {
-                        getConceptCauseOfDeath(uuid);
-                    } else {
-                        mPatientInfoView.cannotMarkDeceased(R.string.mark_patient_deceased_invalid_uuid);
-                    }
+            public void onSuccess(@Nullable String response) {
+                if (response.length() == 36) {
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientPresenter.java:280]: Everything is fine, just add this 36 in constants file. Rest i will run this on studio and if everything works finr, I'll merge

Comment (LuGO0) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientPresenter.java:280]: Sure @rishabh-997 will do it 👍

**Comment by LuGO0** [comment]
@rishabh-997 I added 36 in the application constants 👍

**Review by rishabh-997 — APPROVED** [review]

**Pull request merged**